src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -90,7 +117,11 @@ public int getMinFloorSettings() {
 
     public void setMinFloorSettings(int minFloorSettings) {
         requireNonNull(minFloorSettings);
-        this.minFloorSettings = minFloorSettings;
+        if (minFloorSettings < MAX_ALLOWED_FLOORS && minFloorSettings > MIN_ALLOWED_FLOORS) {
```

Comment:
Should it be `<=` instead of `<` and `>=` instead of `>`?
Same for `setMaxFloorSettings()`.